diff --git a/index.d.ts b/index.d.ts
index af84295c58ad738cc1a3e96aebcd8408841e1003..b998f14c5f0738a229f261baed5e298f0d840021 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -386,7 +386,7 @@ declare namespace request {
     type Cookie = tough.Cookie;

     interface CookieJar {
-        setCookie(cookieOrStr: Cookie | string, uri: string | Url, options?: tough.CookieJar.SetCookieOptions): void;
+        setCookie(cookieOrStr: Cookie | string, uri: string | Url, options?: tough.SetCookieOptions): void;
         getCookieString(uri: string | Url): string;
         getCookies(uri: string | Url): Cookie[];
     }
