diff --git a/index.d.ts b/index.d.ts
index accb78cdf40758bfcd2ee63b7b05d8f863625c4a..943961ae1438bca0d9895f5e77e510233686e2a8 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,7 +1,6 @@
 /// <reference types="node" />
 
 import type { createPrivateKey, createPublicKey, KeyObject } from "crypto";
-import type { StringValue } from "ms";
 
 export class JsonWebTokenError extends Error {
     inner: Error;
@@ -40,8 +39,8 @@ export interface SignOptions {
      */
     algorithm?: Algorithm | undefined;
     keyid?: string | undefined;
-    expiresIn?: StringValue | number;
-    notBefore?: StringValue | number | undefined;
+    expiresIn?: string | number;
+    notBefore?: string | number | undefined;
     audience?: string | string[] | undefined;
     subject?: string | undefined;
     issuer?: string | undefined;
