diff --git a/node_modules/strip-ansi/index.js b/node_modules/strip-ansi/index.js
index 5f3b5d5..e7b631a 100644
--- a/node_modules/strip-ansi/index.js
+++ b/node_modules/strip-ansi/index.js
@@ -1,4 +1,6 @@
 'use strict';
-const ansiRegex = require('ansi-regex');
+const ansiRegexImport = require('ansi-regex');
+const ansiRegex = typeof ansiRegexImport === 'function'
+  ? ansiRegexImport
+  : ansiRegexImport.default;
 
-module.exports = string => typeof string === 'string' ? string.replace(ansiRegex(), '') : string;
+module.exports = string =>
+  typeof string === 'string' && ansiRegex ? string.replace(ansiRegex(), '') : string;
